doc/source/getting_started/first_steps.rst

```diff
@@ -1,8 +1,9 @@
-###############
-Getting started
-###############
+.. _first_steps:
+
+##################
+First steps
+##################
 
-Welcome to the ODL getting started guide.
 This guide is intended to give you a simple introduction to ODL and how to work with it.
 If you need help with a specific function you should look at the `ODL API reference <https://odlgroup.github.io/odl/odl.html>`_.
 
```

Comment:
Perhaps not. The normal equation variant is perhaps less common and appears more useful since it's more general. There's also the danger that users think `conjugate_gradient` makes no assumptions. I think this leads more into the right direction, even if it's not necessary.